app/components/award_details/view.rb

```diff
@@ -11,7 +11,11 @@ def initialize(trainee)
     end
 
     def award_date
-      date_for_summary_view(trainee.awarded_at)
+      if trainee.awarded_at.present?
+        date_for_summary_view(trainee.awarded_at)
+      elsif trainee.recommended_for_award?
+        "Waiting for award - met standards on #{date_for_summary_view(trainee.recommended_for_award_at)}"
```

Comment:
I remember us trying to put text in components into translations. I'm not sure how religiously that was followed but is it worth doing that here?

Reply:
Done